pkg/machines/hostvmslist.jsx

```diff
@@ -176,8 +176,10 @@ export const StateIcon = ({ state, config, valueId, extra }) => {
             className: 'pficon pficon-ok icon-1x-vms',
             title: _("The VM is suspended by guest power management."),
         },
+        creating: { className: 'pficon pficon-pending icon-1x-vms', title: _("The VM is being created.") },
```

Comment:
@mareklibra what is the purpose of the className icons?

Reply:
Thanks for pointing this out.

It's leftover, they should be removed within https://github.com/cockpit-project/cockpit/pull/7595 . The icons are not used anymore.
This needs a follow-up to clean